### PR TITLE
Try jw cms content store rework

### PR DESCRIPTION
**Pull request by ReallySmall**

### CODE REVIEW

File: futurenhs.content.api/Umbraco9ContentApi.Umbraco/appsettings.json [hunk 1]
```diff
@@ -7,8 +7,8 @@
       "Microsoft.Hosting.Lifetime": "Information"
     },
     "TableStorageConfiguration": {
-      "ConnectionString": "",
-      "TableName": ""
+      "ConnectionString": "DefaultEndpointsProtocol=https;AccountName=cdslocaldevstorage;AccountKey=UYeJXwqzx9C3q07+xwy9CHwcM+LKwl4X9hOlw4FpWeGlQZIvlLkzxaUyhh/M9+f1jvor0NMZ9N9gn5lzr2R87A==;EndpointSuffix=core.windows.net",
```
Comment (Tim-Hunt-CDS) [inline futurenhs.content.api/Umbraco9ContentApi.Umbraco/appsettings.json:10]: should be in secrets

File: futurenhs.app/services/getCmsPageContent/index.ts [hunk 2]
```diff
@@ -47,6 +48,8 @@ export const getCmsPageContent = async (
 
     const { headers, ok, status, statusText } = apiMeta
 
+    console.log(apiData);
```
Comment (andy-smith-cds) [inline futurenhs.app/services/getCmsPageContent/index.ts:51]: a stray console log

File: futurenhs.app/components/ContentBlockWrapper/index.tsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+                id="dialog-discard-cms-block"
+                isOpen={isDeleteModalOpen}
+                text={{
+                    cancelButton: 'Cancel',
+                    confirmButton: 'Yes, discard',
+                }}
+                cancelAction={handleDeleteCancel}
+                confirmAction={handleDeleteConfirm}
+            >
+                <h3>Entered Data will be lost</h3>
```
[the remark this answers is not in the document]
Comment (ReallySmall) [reply futurenhs.app/components/ContentBlockWrapper/index.tsx]: Resolved

File: futurenhs.app/services/getCmsPageContent/index.ts (hunk 2 above)
Comment (ReallySmall) [reply futurenhs.app/services/getCmsPageContent/index.ts]: Removed

**Pull request merged**